.github/workflows/build-ucxx.yaml

```diff
@@ -1,4 +1,4 @@
-name: build
+name: Build Dask-CUDA for UCXX
```

Comment:
The current name is a duplicate of the one in the "default" `build.yaml`, which causes a clash in the "Actions" panel and workflow name matching.

![Screenshot 2025-05-20 at 22 39 24](https://github.com/user-attachments/assets/87c8bd31-db09-49bd-bee9-98dbccff3f24)

Reply:
ah yeah! This is a good change, thanks for the explanation.